sessions/recap-project-5/art-gallery-app/README.md

```diff
@@ -49,228 +43,222 @@ Download a [Favorite Icon](https://lucide.dev/icons/) and save it in an "assets"
 
 🚀 Project Deployment to Vercel is required: Please follow [these instructions](https://github.com/neuefische/web-protocol-template/blob/main/docs/deployment-vercel.md) to deploy the project on Vercel.
 
-## User Stories
+## Tasks
+
+In this app, users will have access to a **Spotlight**, **Gallery**, **Favorite**, and **Detail** page.
 
-### User Story 1: List of all Art Pieces
+### 1. Spotlight Page
+
+This page will serve as the app's homepage. A random art piece image, along with its artist’s name, will be displayed as a spotlight feature.
 
 #### Value Proposition
 
 **As an** art enthusiast
 
-**I want to** see a list of all art pieces
+**I want to** see a spotlight art piece
 
-**so that** I can get an overview of all art pieces.
+**so that** I can get inspirational highlights.
 
 #### Acceptance Criteria
 
-- [ ] All art pieces are displayed as a list
-- [ ] Each art piece's image is displayed
-- [ ] Each art piece's title is displayed
-- [ ] Each art piece's artist is displayed
+- [ ] One art piece is randomly selected as a spotlight feature.
+- [ ] The image of the selected art piece is displayed.
+- [ ] The artist's name for the spotlighted art piece is displayed.
 
 #### Tasks
 
-- [ ] Fetch all art pieces with `SWR` in `pages/index.js`
-- [ ] Create the component `ArtPieces` to render a list
-- [ ] `ArtPieces` props: `pieces`
-- [ ] Create the component `ArtPiecePreview`
-- [ ] `ArtPiecePreview` props: `image`, `title`, `artist`
+- [ ] Write a function to select a random art piece. [Refer to this resource](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random#examples).
+- [ ] Create a new component for the Spotlight page.
+- [ ] Write tests for the Spotlight component.
```

Comment:
do we really want to add tests in this recap project? It makes sense since they learn about it prior, but maybe its a bit too much and they get stuck?

Reply:
I wrote with all the test that we have at SP, but we could select certain test for them to do. Select certain components.

Reply:
What do you mean exactly, I didn't understand 😅